gateway/1.4/modules/ROOT/pages/_partials/task-config-proxy-protocol.adoc

```diff
@@ -0,0 +1,101 @@
+//tag::intro1[]
+For use cases where Flex Gateway is deployed behind a load balancer or other proxy service, configure Proxy Protocol to preserve client IP addresses across multiple layers of NAT or TCP proxies.
+
+In Proxy Protocol, the load balancer or other proxy service sends the Proxy Protocol header containing the client address and additional information to Flex Gateway. To learn more about Proxy Protocol, see the https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt[HAProxy specification^].
+
+Enabling Proxy Protocol only enables Flex Gateway to receive the Proxy Protocol header. Your load balancer or other proxy service must support Proxy Protocol to transmit the header.
+
+//end::intro1[]
+
+//tag::icon-table[]
+
+[cols="1a,1a,1a"]
+|===
+|image:install-linux-logo.png[20%,20%,xref="#linux"]
+|image:install-docker-logo.png[25%,25%,xref="#docker"]
+|image:install-kubernetes-logo.png[20%,20%,xref="#kubernetes"]
+
+|<<linux>>
+|<<docker>>
+|<<kubernetes>>
+|===
+
+//end::icon-table[]
+
+//tag::byb[]
+
+== Before You Begin
+
+Before enabling Proxy Protocol for Flex Gateway, complete the following tasks:
+
+* xref:flex-install.adoc[Download Flex Gateway]
+* xref:flex-{page-mode}-reg-run.adoc[Register and Run Flex Gateway]
+
+//end::byb[]
+
+//tag::linux[]
+
+[[linux]]
+== Enable Proxy Protocol for Flex Gateway as a Linux Service
+
+. Create a YAML configuration file in the Flex Gateway configuration directory:
++
+[source,ssh]
+----
+sudo touch /usr/local/share/mulesoft/flex-gateway/conf.d/proxy-protocol-config.yaml
+----
+
+. Update the file with the Proxy Protocol configuration file: 
++
+//end::linux[]
+
+//tag::docker-intro[]
+
+[[docker]]
+== Enable Proxy Protocol for Flex Gateway in a Docker Container
+
+NOTE: If you have already added a volume for a folder with your
+Flex Gateway configuration files, go to the last step.
```

Comment:
```suggestion
Flex Gateway configuration files, skip to the last step.
```

Reply:
I like this wording better, but we have "go" other places in the docs.

Reply:
I prefer skip because it tells users they don't need to look at all the other steps.